core-web/libs/portlets/edit-ema/portlet/src/lib/store/features/editor/withEditor.ts

```diff
@@ -54,9 +54,10 @@ const buildIframeURL = ({ url, params, isTraditionalPage }) => {
     }
 
     // Remove trailing slash from host
-    const host = (params.clientHost || window.location.origin).replace(/\/$/, '');
+    const dotCMSHost = window.location.origin;
```

Comment:
Check if we don't have this in the store, maybe we can do a computed, to use the `WINDOW` injector

Reply:
This function is called inside a compute

Reply:
Done!